pkg/k8sops/generators.go

```diff
@@ -188,6 +192,9 @@ func GenerateStatefulSet(
 	} else {
 		template := cluster.Spec.DataDirVolumeClaimTemplate.DeepCopy()
 		template.ObjectMeta.Name = _dataVolumeName
+		if sc := isolationGroup.StorageClassName; sc != "" {
```

Comment:
Is there anyway to ensure that there is a storage class of that name within the template itself? Do that need to validated?

Reply:
I think for now it's better to let Kube throw the error when it tries to create the statefulset and doesn't find it, otherwise we'd have to add new RBAC permissions to read StorageClasses

Reply:
Fair, I wanted to ensure that it would be clear to user that a storage class doesn't exist.